apps/client/src/widgets/community/components/user-comment-reply/user-comment-reply.css.ts

```diff
@@ -1,12 +1,24 @@
 import { style } from '@vanilla-extract/css';
+import { recipe } from '@vanilla-extract/recipes';
 
 import { themeVars } from '@bds/ui/styles';
 
-export const container = style({
-  display: 'flex',
-  flexDirection: 'column',
-  padding: '0.8rem 0',
-  gap: '0.4rem',
+export const container = recipe({
+  base: {
+    display: 'flex',
+    flexDirection: 'column',
+    padding: '0.8rem 0',
```

Comment:
```suggestion
    padding: '1.2rem 0',
```

피그마 보니까 1.2rem으로 되어 있더라고요.. 수정 부탁드려요 !!

Reply:
헉,,,,, 감사합니다..